vowpalwabbit/recall_tree.cc

```diff
@@ -355,9 +354,19 @@ float train_node(recall_tree& b, single_learner& base, example& ec, uint32_t cn)
   double delta_left = nl * (new_left - old_left) + mc.weight * new_left;
   double delta_right = nr * (new_right - old_right) + mc.weight * new_right;
   float route_label = delta_left < delta_right ? -1.f : 1.f;
-  float imp_weight = fabs((float)(delta_left - delta_right));
 
-  ec.l.simple = {route_label, imp_weight, 0.};
+  // Bug?
+  // Notes: imp_weight does not seem to be used since it was not set to
+  // ec.weight before.  ec.l.simple.weight was not used in gd.
+  // float imp_weight = fabs((float)(delta_left - delta_right));
+
+  ec.l.simple = {route_label, VW::UNUSED_1, VW::UNUSED_0};
+  // Bug?  
```

Comment:
is this a bug? should it be documented?

Reply:
Let's ask John during PR review.  A can be raised in vw repo if needed.